modules/tide_media/tide_media.install

```diff
@@ -81,3 +81,35 @@ function tide_media_update_10005() {
   $config->set('track_enabled_target_entity_types', $result);
   $config->save();
 }
+
+/**
+ * Adds secure file widget to entity browser.
+ */
+function tide_media_update_10006() {
+  if (\Drupal::moduleHandler()->moduleExists('tide_media_secure_files')) {
```

Comment:
More a comment than anything, but would we normally do this like this or would we load the config from `config/install` and merge it in?